content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,299 @@
+individual payslip.
+
+.. _payroll/new-payslip:
+
+Create a new payslip
+--------------------
+
+A new payslip can be created by clicking the :guilabel:`Create` smart button in the top left.
+
+.. image:: payslips/create-payslip.png
+   :align: center
+   :alt: Click the Create button to make a new payslip.
+
+An empty payslip form will appear. Fill out the payslip information.
+
+Required fields
+~~~~~~~~~~~~~~~
+
+.. image:: payslips/new-payslip.png
+   :align: center
+   :alt: The necessary fields for a new payslip.
+
+- :guilabel:`Employee`: Select the employee from the drop down list. Several fields may be populated
+  after making an Employee selection, typically the *Contract*, *Structure*, and *Payslip Name*
+  fields.
+- :guilabel:`Period`: Using the drop down, select the start and end date for the payslip. In the
+  drop down, use the left and right arrows to select the month, and click on the day to select the
+  date.
+- :guilabel:`Contract`: Using the drop down menu, select the contract for the employee. Only the
+  available correspoding contracts for the selected employee will appear as options.
+- :guilabel:`Structure`: Using the drop down menu, select the salary structure type. Only the
+  available correspoding structures for the selected contract for the specific employee will appear
+  as options.
+- :guilabel:`Payslip Name`: Type in the name for the payslip. The name should be short and
+  descriptive, such as "April 2023".
+
+.. image:: payslips/new-payslip-tab.png
+   :align: center
+   :alt: The necessary fields for a new payslip in the Acounting Information tab.
+
+- :guilabel:`Company`: In the *Accounting Information* tab, select the company the payslip applies
+  to from the drop down.
+- :guilabel:`Salary Journal`: In the *Accounting Information* tab, enter the salary journal the
+  payment will be reflected in in the *Accounting* applicaiton.
+
+.. important::
+   It is recommended to check with the accounting department to ensure any entry that affects the
+   *Accounting* application is correct.
+
+Optional fields
+~~~~~~~~~~~~~~~
+
+ - :guilabel:`Reference`: Any note or reference message for the new entry can be entered here.
+ - :guilabel:`Company Car`: If applicable, select the company car from the drop down.
+ - :guilabel:`Worked Days`: In the *Worked Days & Inputs* tab, the entry under :guilabel:`Worked
+   Days` (the type, description, number of days, number of hours, and amount) will be automatically
+   filled in based off what was entered for the period, contract, and structure.
+ - :guilabel:`Salary Computation`: This section will be automatically filled in after the
+   :guilabel:`Compute Sheet` smart button is clicked. This will display all the wages, deductions,
+   taxes, etc for the entry.
+ - :guilabel:`Batch Name`: Select the payslip batch this new payslip should be added to from the
+   drop down.
+ - :guilabel:`Made Payment Order`:
```

Comment:
OMG I can't believe that was left blank! I have an open question on this to 3 different people, waiting for them to get back to me.

Reply:
OK, I just checked with the PO and the support person- this is an outdated item that does not do anything! They are going to work now to remove this box since it has been doing nothing for several versions before 14.

Reply:
Apparently Make Payment Order doesn't do anything and hasn't in a while! They are going to remove this from the app, so I'll delete this line.